gaia-button.js

```diff
@@ -24,6 +24,24 @@ module.exports = component.register('gaia-button', {
     this.setupShadowRoot();
     this.setAttribute('role', 'button');
     this.tabIndex = 0;
+    if (this.keyboardEnabled) {
```

Comment:
Well, that depends. There is no overhead to bind all of these events. How do yo feel?